module/skill/GetUp/data/config/webots/GetUp.yaml

```diff
@@ -11,11 +11,9 @@ scripts:
     - "StandUpBack.yaml"
     - "Stand.yaml"
   getup_left:
-    - "RollLeft.yaml"
     - "StandUpFront.yaml"
     - "Stand.yaml"
   getup_right:
-    - "RollRight.yaml"
```

Comment:
should really put a comment in or make an issue

Reply:
I think they tend to get up fine without rolling right/left in WeBots

Reply:
the get up scripts typically roll them onto a side but I don't think they have time to try getting up twice before the auto ref  penalises?